properties.go

```diff
@@ -96,3 +110,20 @@ func parseStringArray(value interface{}) []string {
 	}
 	return list
 }
+
+func parseKeyValueArray(value interface{}) map[string]string {
+	locations, ok := value.([]interface{})
+	if !ok {
+		return map[string]string{}
+	}
+	mappedLocations := make(map[string]string)
```

Comment:
this variable needs a name change as it's not necessary related to mapped locations ;-)
`keyValueArray` might work.

Reply:
my bad